dapp/js/containers/Pages/CampaignDeployer/Component.jsx

```diff
@@ -28,6 +28,16 @@ export default class Deployer extends Component {
 
   componentDidMount() {
     this.props.setAccount(web3.eth.accounts[0]);
+    //set default account to user current account if they are not set
+    let campaignValues = Object.assign({}, this.props.campaignValues);
+    let accountTypes = ['escapeCaller', 'securityGuard', 'donor', 'recipient']
+    for(let i=0; i<accountTypes.length; i++) {
+        if(!campaignValues[accountTypes[i]]){
+            campaignValues[accountTypes[i]] = web3.eth.accounts[0];
+        }
+    }
+    this.props.updateCampaignValues(campaignValues);
```

Comment:
setAccount does the re-render too. As the needed value is web3.eth.accounts[0] and here is the first time to query this value, so I had to put these lines here. 
To hardcode it you should have loaded accout[0] first  and currently I don't think the system is doing it in that way. 
The ideal scene IMO is to load web3 object as a state variable in the first page laoding and re-render the whole page whenever user change their account in their metamask.
Any suggestions for this issue?


One more issue, I am expeirencing(I experienced once in one of my demo too) that sometimes metamask does not load completely when the page is loaded and your currently active account may not be loaded either unless you refresh you page a couple of times. What I did is to load bundle.js after the HTML Is fully loaded(a little ugly)